huawei-wmi.c

```diff
@@ -922,6 +924,79 @@ static void huawei_wmi_power_unlock_exit(struct device *dev)
 		device_remove_file(dev, &dev_attr_power_unlock);
 }
 
+/*Fan speed*/
+
+static int huawei_wmi_fan_speed_get(u8 num, int *rpm)
+{
+	u8 ret[0x100] = { 0 };
+	int err;
+	
+	union hwmi_arg arg;
+	arg.cmd = FAN_SPEED_GET;
+	arg.args[2] = num;
+
+	err = huawei_wmi_cmd(arg.cmd, ret, 0x100);
```

Comment:
What exactly is the `0x100` constant? If it's something meaningful, then maybe `#define` it?

Reply:
`0x100` is the buffer size the HWMI interface allocates. I think it would be a good idea to `#define` it somewhere